modules/ecs-service/README.md

```diff
@@ -7,21 +7,144 @@ The following requirements are needed by this module:
 
 - <a name="requirement_aws"></a> [aws](#requirement\_aws) (>= 4.36)
 
-## Providers
+## Sample Usage
 
-The following providers are used by this module:
+```hcl
+module "example" {
 
-- <a name="provider_aws"></a> [aws](#provider\_aws) (>= 4.36)
 
-## Resources
+	 source  = "github.com/Cyber4All/terraform-cyber4all-catalog//modules/<REPLACE_WITH_MODULE>?ref=v<REPLACE_WITH_VERSION>"
 
-The following resources are used by this module:
 
-- [aws_cloudwatch_log_group.logs](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/cloudwatch_log_group) (resource)
-- [aws_ecs_service.service](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ecs_service) (resource)
-- [aws_ecs_task_definition.task](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ecs_task_definition) (resource)
-- [aws_service_discovery_service.registry](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/service_discovery_service) (resource)
+	 # --------------------------------------------
+	 # Required variables
+	 # --------------------------------------------
 
+
+	 # ARN of an ECS cluster.
+	 cluster_arn  = string
+
+
+	 # The ID of the namespace to use for DNS configuration.
+	 dns_namespace_id  = string
+
+
+	 # The image used to start a container. This string is passed directly to the Docker daemon. Images in the Docker Hub registry are available by default. You can also specify other repositories with either `repository-url/image:tag` or `repository-url/image@digest`. Up to 255 letters (uppercase and lowercase), numbers, hyphens, underscores, colons, periods, forward slashes, and number signs are allowed.
+	 image  = string
+
+
+	 # Name that will associate all resources.
+	 service_name  = string
+
+
+	 # --------------------------------------------
+	 # Optional variables
+	 # --------------------------------------------
+
+
+	 # The hard limit of CPU units to present for the task. For tasks that use the Fargate launch type (both Linux and Windows containers), this field is required. 
+	 container_cpu  = number
+
+
+	 # The amount (in MiB) of memory to present to the container. [container_definition_memory](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definition_parameters.html#container_definition_memory)
+	 container_memory  = number
+
+
+	 # The port number on the container that's bound to the user-specified or automatically assigned host port.
+	 container_port  = number
+
+
+	 # Must be set to either `X86_64` or `ARM64`; see cpu architecture.
```

Comment:
This is the cpu architecture? You mean aws documentation?

Reply:
Oh I can actually get rid of that last part.

Reply:
Oh wait ignore ecs-service changes because that is just automated documentation of our old imlementations. Just look at ecs-cluster stuff.